smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/AddBaseServiceExceptionClass.java

```diff
@@ -67,6 +67,9 @@ private void writeAdditionalFiles(
                                     TypeScriptDependency.AWS_SMITHY_CLIENT.packageName);
                             writer.addImport("ServiceExceptionOptions", "__ServiceExceptionOptions",
                                     TypeScriptDependency.AWS_SMITHY_CLIENT.packageName);
+                            // Export ServiceException information to allow
+                            //      documentation inheritance to consume their types
+                            writer.write("export { __ServiceException, __ServiceExceptionOptions }\n");
```

Comment:
Was wondering if this should be the following?

```suggestion
                            // Export ServiceException information to allow
                            //      documentation inheritance to consume their types
                            writer.write("export { __ServiceException, __ServiceExceptionOptions }?");
```